src/App/App.js

```diff
@@ -1,35 +1,47 @@
 import './App.scss';
 import Nav from '../Nav/Nav';
 import React, { Component } from 'react';
-import movieData from '../movieData';
+//import movieData from '../movieData';
```

Comment:
Don't merge commented out code. If you don't need it then you don't need it, it just looks sloppy this way.